### PR TITLE
Added overlay for deletion of referenced pages

### LINKED ISSUE #877: Page deletion should be confirmed if page is referenced
`Uncaught PHP Exception PHPCR\ReferentialIntegrityException: "HTTP 409: 922b6b02-fe82-4b87-8b68-0b45c8bd9fb8 cannot be removed because it is still being referenced"`

### PR DESCRIPTION
**Pull request by danrot**
Has to be merged together with https://github.com/sulu-io/sulu-standard/pull/537 and https://github.com/massiveart/husky/pull/570

__tasks:__

- [x] test coverage
- [x] gather feedback for my changes

__informations:__

| q                | a
| ---------------- | ---
| Fixed tickets    | fixes #877 
| BC breaks        | none
| Documentation PR | none

### CODE REVIEW

File: src/Sulu/Bundle/ContentBundle/Controller/NodeController.php [hunk 1]
```diff
@@ -469,12 +470,39 @@ public function deleteAction(Request $request, $uuid)
     {
         $language = $this->getLanguage($request);
         $webspace = $this->getWebspace($request);
+        $force = $this->getBooleanRequestParameter($request, 'force', false, false);
+
+        $references = array_filter(
```
Comment (chirimoya) [inline src/Sulu/Bundle/ContentBundle/Controller/NodeController.php]: Only load references if `$force` is `false`

File: src/Sulu/Bundle/ContentBundle/Resources/public/js/model/content.js [hunk 2]
```diff
@@ -44,8 +44,18 @@ define([
...
         },
 
-        fullDestroy: function(webspace, language, options) {
-            options = _.defaults((options || {}), {url: this.urlRoot + (this.get('id') !== undefined ? '/' + this.get('id') : '') + '?webspace=' + webspace + '&language=' + language});
+        fullDestroy: function(webspace, language, force, options) {
+            options = _.defaults(
+                (options || {}),
+                {
+                    url: this.urlRoot
+                        + (this.get('id') !== undefined ? '/'
```
Comment (chirimoya) [inline src/Sulu/Bundle/ContentBundle/Resources/public/js/model/content.js]: If `this.get('id') === undefined` there is nothing to delete. Throw error or return early?

**Comment by chirimoya** [comment]
else ack

**Comment by danrot** [comment]
@chirimoya fixed

**Pull request merged**